README.md

```diff
@@ -33,7 +33,13 @@ Please send hiring@truss.works a link to a public git repository
 how to build and run it. Your code will be run on either macOS 10.15
 or Ubuntu 16.04 LTS, your choice.
```

Comment:
```suggestion
or Ubuntu 16.04 LTS, your choice. If you choose the browser solution, your code will be run in Firefox, Chrome, or Safari.
```

I propose listing Safari last b/c some folks may not have MacOS which makes testing in Safari difficult.